src/test/allocator_tests.cpp

```diff
@@ -144,6 +144,10 @@ class TestLockedPageAllocator: public LockedPageAllocator
                 *lockingSuccess = true;
             }
 
+#ifdef _MSC_VER
+// Disable MSVC warning: C4312: 'reinterpret_cast': conversion from 'int' to 'void *' of greater size
+#pragma warning(disable: 4312)
+#endif
```

Comment:
Can't this be solved by `return reinterpret_cast<void*>(uint64_t{0x08000000 + (count<<24)});`?